dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -217,15 +223,48 @@ public void start(WorkerNetAddress address) throws IOException {
     // the heartbeat is only used to notify the aliveness of this worker, so that clients
     // can get the latest worker list from master.
     // TODO(bowen): once we set up a worker discovery service in place of master, remove this
+    /*
```

Comment:
remove commented code?

Reply:
add back to follow old workflow logic if membership module not enabled by default